lh-test/src/main/java/io/littlehorse/test/AbstractVerifier.java

```diff
@@ -30,15 +35,20 @@ public AbstractVerifier(
     public void start() {
         try {
             lhClientTestWrapper.registerWfSpec(workflow);
-            WfSpec wfSpec = Awaitility.await().until(() -> lhClientTestWrapper.getWfSpec(workflow), Objects::nonNull);
+            WfSpec wfSpec = Awaitility.await()
```

Comment:
Is there any configurabilty to determine how often it polls? Like can we make it poll every 50 milliseconds?

Reply:
we can use these methods for that:

```
Awaitility.setDefaultPollInterval(10, TimeUnit.MILLISECONDS);
Awaitility.setDefaultPollDelay(Duration.ZERO);
```